### PR TITLE
:heavy_check_mark:  Math Magicians: Components

### PR DESCRIPTION
**Pull request by mahabubx7**
### :heavy_check_mark:  Math Magicians: Components
- [x] No linters error.
- [x] Maintained git-flow.
- [x] Deleted all unnecessary boilerplate codes
- [x] Added `src/components` directory
- [x] Added `Calculator` component
- [x] Designed the calculator UI according to the instructions.
- [x] Linked the `Calculator.js` component in the `App.js` root level.
- [x] Tested in browser (no console error)
- [x] Added at least one child component `Button Component`

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    </div>
+  );
+}
+
+function Button({ content, className }) {
+  return (
+    <button type="button" className={className}>{content}</button>
+  );
+}
+
```
Comment (levy002) [inline src/components/Calculator.js]: - [x] You did a great job of creating the `Button` component but I highly recommend you move this `Button` component to its own file in order to keep your repo clean and professional. Make sure you import this component where it needs to be used.

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+// import logo from './logo.svg';
+// import './App.css';
```
Comment (levy002) [inline src/App.js]: - [x] Kindly make sure to remove all comments in your codebase in order to keep your repo clean and professional.

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+// import './index.css';
```
Comment (levy002) [inline src/index.js]: - [x] Kindly make sure to remove all comments in your codebase in order to keep your repo clean and professional.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @mahabubx7   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors :heavy_check_mark: 
- Well-detailed PR summary :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (rloterh) [reply src/components/Calculator.js]: well resolved! ✔️

File: src/App.js (hunk 2 above)
Comment (rloterh) [reply src/App.js]: well resolved! ✔️

File: src/index.js (hunk 3 above)
Comment (rloterh) [reply src/index.js]: well resolved! ✔️

**Comment by mahabubx7** [comment]
@rloterh I think you are supposed to make a review comment and then also approve or request changes.
but sorry to say, you directly requested another change! What I'm supposed to do?

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mahabub,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Properly implemented React components!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly remove the Live demo if this app is not yet deployed.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rloterh** [comment]
> @rloterh I think you are supposed to make a review comment and then also approve or request changes. but sorry to say, you directly requested another change! What I'm supposed to do?

Hi @mahabubx7 

Your project is complete and approved!
Cheers! 😃

**Pull request merged**